jobbergate-cli/jobbergate_cli/subapps/job_submissions/tools.py

```diff
@@ -43,7 +42,6 @@ def create_job_submission(
 
     if client_id is None:
         client_id = jg_ctx.persona.identity_data.client_id
-    validate_client_id(jg_ctx, client_id)
```

Comment:
I think **if** the client is passed then it should be checked, otherwise, life goes on.

Reply:
We can't validate the client with stand-alone Jobbergate. This relies on the cluster-api, which won't be available for users that are only using Jobbergate.

We can approach this in different ways later, but for now, to allow Jobbergate to be a stand-alone platform, it needs to be removed.